xdsmatcher/go.mod

```diff
@@ -3,14 +3,9 @@ module github.com/envoyproxy/go-control-plane/xdsmatcher
 go 1.15
```

Comment:
Might be a good idea to bump this to whatever is at the root go.sum too. We can do that later though

Reply:
I think it worth another PR Indeed. As 1.19 has been published, it might even be an opportunity to upgrade go version to 1.17 for all thé modules